README.md

```diff
@@ -129,7 +129,7 @@ base. Your initial kustomization.yaml could be as simple as:
 
 ```yaml
 resources:
-  - https://github.com/statnett/image-scanner-operator//config/default?ref=vMAJOR.MINOR.PATCH
+  - https://github.com/statnett/image-scanner-operator?ref=vMAJOR.MINOR.PATCH
```

Comment:
```suggestion
  - https://github.com/statnett/image-scanner-operator?ref=v0.0.0 # x-release-please-version
```

Reply:
Nice suggestion! I'll do this, and also make release-please aware of it.